api/views/login-test.ejs

```diff
@@ -1,4 +1,4 @@
-<form action="/api/login">
+<form action="/api/ingest/login">
```

Comment:
Rather than having the login routes under /api/ingest, it probably makes more sense to have them in the root of /api... I'm just not entirely sure how best to set that up, or exactly what role the setupPassport function has. Do we need to run setupPassport on all of our routers, or is it a one-time thing that should be called from app.js somehow? We should talk to @crhallberg about this before merging this work.